src/helpers.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import loadScript from 'load-script';
+
+/* global window CKEDITOR */
+
+let promise;
+
+export function getEditorNamespace( editorURL ) {
+	if ( editorURL.length < 1 ) {
```

Comment:
>What if I don't care about URL because I know that I already attached CKEditor externally? Will I need to provide some fake URL to get around this condition?

In this case default value will be provided by framework, yet if `CKEDITOR` namespace is present no scripts are loaded.

I'll slightly rework it tho, so it won't throw when `editorUrl` is overridden with empty string, but `CKEDITOR` namespace is present.

> we should throw error where there is actually no CKEditor instance loaded by loadScript function?

👍